app/matplotlib.ipynb

```diff
@@ -194,21 +294,45 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 10,
-   "outputs": [],
    "source": [
     "labels = [\"Product A\", \"Product B\", \"Product C\", \"Product D\"]\n",
     "market_share = [30, 20, 15, 35]\n",
     "\n",
-    "# write your code here"
+    "plt.pie(market_share, labels=labels, autopct='%1.1f%%', startangle=90)\n",
```

Comment:
The pie plot is missing the command to display the plot. You should add `plt.show()` at the end to display the plot.